### PR TITLE
[WIP]I'm trying to make a semantic UI stylesheet/theme, not quite sure how…

### PR DESCRIPTION
**Pull request by dominikkawka**
… it works and can't wrap my head around it.   I am only using index for testing. 

# Description

Trying to make a theme with semantics
## What

I want to make it so that we can edit the default values of colours

## Why

So we can use the colours that we want

## How

Making a semantic stylesheet and linking it to pages

## Verification Steps

Opening/refreshing a browser

# Checklist:

- [ ] This pr is linked to the related trello card
- [x] My code follows the style guidelines of this project
- [ ] I have requested a code review from (at least) two team members
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] This change has been verified by another team member

### CODE REVIEW

**Comment by dominikkawka** [comment]
@laurafitzgerald @ciaranRoche @dimitraz @JameelB @judyobrienie

**Comment by laurafitzgerald** [comment]
Hi @dominikkawka. The way we currently use Semantic we can't use themes. To use themes the Semantic package needs to be installed rather than just linked as we have it. I did some investigations into this myself on a [site](https://github.com/laurafitzgerald/lisafitzgerald-therapies) I was tinkering with. I wouldn't worry about this too much though as the styling we have on the color scheme on the site isn't too heavy right now. The main thing we are using Semantic for is the layout. I think this is a good idea to investigate this but I would suggest waiting to see if we have some time at the end of the sprint. There's still a few cards left that we want to get completed first.

**Comment by maciejmarchel12** [comment]
So how can we change colours then without having the themes?

File: docs/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,29 +0,0 @@
-# Description
```
Comment (laurafitzgerald) [inline docs/PULL_REQUEST_TEMPLATE.md]: Why you try to delete my fiel :P

Comment (maciejmarchel12) [reply docs/PULL_REQUEST_TEMPLATE.md]: I think something went wrong and instead of the contact html being just uploaded more files were added

**Pull request closed without merging**